### DISCUSSION TITLE
The integration with wagmi on the development documentation is outdated and needs to be updated to be compatible with wagmi 2.x.

### DISCUSSION THREAD

**Discussion by naxida-eth** [phantom/docs#259] [Help & Support]
### Summary

https://docs.phantom.app/library-integrations/wagmi

The integration with wagmi on the development documentation is outdated and needs to be updated to be compatible with wagmi 2.x.

The version of wagmi shown in the official example is wagmi 1. It needs to be updated to version 2.x.

### Example

https://github.com/0xproflupin/phantom-sandboxes/blob/main/package.json

### Steps to Reproduce

see: https://github.com/0xproflupin/phantom-sandboxes/blob/main/package.json

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
@naxida-eth Thank you for letting us know! I will look into the updates.

**Reply by Pelumi527** [reply]
@naxida-eth where you able to integrate the `phantom-wagmi-connector` with the latest the wagmi `2.x`

**Reply by IGMontero** [reply]
Any updates on this?

**Reply by Pelumi527** [reply]
You can try this 
`const phantomConnectorInjected = injected({
  target: "phantom"
})`

`const connectors = [
  phantomConnectorInjected,
];`


```export const config = createConfig({
  chains: [mainnet, sepolia, base, polygon, arbitrum],
  connectors,
  transports: {
    [mainnet.id]: http(),
    [sepolia.id]: http(),
    [base.id]: http(),
    [polygon.id]: http(),
    [arbitrum.id]: http() 
  },
  ssr: false
});```
